File: WinForms/CS/LayoutMainDemo/Modules/LayoutPrinting.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Drawing;
using System.Data;
using System.Linq;
using System.Text;
using System.Windows.Forms;
using System.Data.OleDb;
using DevExpress.Internal;
using DevExpress.XtraLayout.Demos.Modules;

namespace DevExpress.XtraLayout.Demos {
    public partial class LayoutPrinting : DragDropLayoutControl {
        public LayoutPrinting() {
            InitializeComponent();
        }
        void InitData() {
            string DBFileName = DataDirectoryHelper.GetDataFile("nwind.xml");
            if(DBFileName != "")
                dsCategories.ReadXml(DBFileName, XmlReadMode.InferSchema);
        }
        void LayoutPrinting_Load(object sender, EventArgs e) {
            if(!DesignMode) InitData();
        }
    }
}

File: WinForms/VB/LayoutMainDemo/Modules/LayoutPrinting.vb
Imports System
Imports System.ComponentModel
Imports System.Drawing
Imports System.Data
Imports DevExpress.Internal
Imports DevExpress.XtraLayout.Demos.Modules

Namespace DevExpress.XtraLayout.Demos

    Public Partial Class LayoutPrinting
        Inherits DragDropLayoutControl

        Public Sub New()
            InitializeComponent()
        End Sub

        Private Sub InitData()
            Dim DBFileName As String = DataDirectoryHelper.GetDataFile("nwind.xml")
            If Not Equals(DBFileName, "") Then dsCategories.ReadXml(DBFileName, XmlReadMode.InferSchema)
        End Sub

        Private Sub LayoutPrinting_Load(ByVal sender As Object, ByVal e As EventArgs)
            If Not DesignMode Then InitData()
        End Sub
    End Class
End Namespace
